web/src/services/Selector.service.ts

```diff
@@ -16,12 +16,15 @@ const getValue = (value: string): string => {
   return `"${value}"`;
 };
 
-const selectorRegex = /span\[(.*)\]/i;
+const selectorRegex = /span\[(.*)]/i;
 const nthChildNumberRegex = /\((.*)\)/i;
 const operationRegex = /([=]+|contains)/;
 
-const getFilters = (selectors: TSpanSelector[]) =>
-  selectors.map(({key, operator, value}) => `${key} ${operator} ${getValue(value)}`);
+const getFilters = (selectors: TSpanSelector[]) => {
+  return selectors.map(
+    ({key, operator, value}) => `${key}${operator === 'contains' ? ` ${operator} ` : operator}${getValue(value)}`
```

Comment:
We can always have the spaces between for the operator so I'm not sure that we need to check for `contains` directly.

Reply:
Let's roll back the changes made around the selectors and we should be good to merge!